lib/taurus/core/tango/tangoattribute.py

```diff
@@ -603,24 +605,36 @@ def _subscribeEvents(self):
         self.__subscription_event = threading.Event()
         attr_name = self.getSimpleName()
 
+        if self._ignore_events:
+            self.__chg_evt_id = -1
+            self._activatePolling()
+            return
+
         try:
             self.__subscription_state = SubscriptionState.Subscribing
-            self.__chg_evt_id = self.__dev_hw_obj.subscribe_event(
-                attr_name, PyTango.EventType.CHANGE_EVENT,
-                self, [])  # connects to self.push_event callback
-
+            self._call_dev_hw_subscribe_event(False)
```

Comment:
I am not sure about this change. Note that the implementation of `_call_dev_hw_subscribe_event` handles all exceptions, whereas the original code may raise exceptions on errors, which would in turn trigger the code in the except clause of this try-except block.
It seems to me that the new implementation makes that the except clause is never reached.